README.md

```diff
@@ -17,113 +17,201 @@ The Sentry extension renders a `View logs in Sentry` next to error throwing stat
 
 ## Language support
 
-To work, the Sentry Sourcegraph extension must know how to recognize instances of error handling and/or exception throwing for each language. The first version will support:
+The Sentry Sourcegraph extension uses common error handling and/or exception throwing patterns specific to each language to identfy Sentry relevant lines of code. The following languages are currently supported:
 
-- TypeScript
-- Go
-- JavaScript
-- Python
-- Java
+- TypeScript ( e.g. `throw new Error()`)
```

Comment:
Last nit: merge TS and JS because this makes it look like `throw` is not supported for JS, and `console` is not supported for TS.

```suggestion
- TypeScript, JavaScript ( e.g. `throw new Error()`, `console.error()`)
```